From: Punit Agrawal <punitagrawal@gmail.com>
Date: Sat, 7 Jan 2017 11:00:02 +0000
Subject: gtags: Fix lintian warning with gtags manpage

---
 gtags/manual.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtags/manual.in b/gtags/manual.in
index 2a2b2b4..43e0347 100644
--- a/gtags/manual.in
+++ b/gtags/manual.in
@@ -59,7 +59,7 @@
 		@var{bindir}, @var{libdir}, @var{datadir}, @var{localstatedir} and @var{sysconfdir}.
 	@item{@option{-d}, @option{--dump} @arg{tag-file}}
 		Dump a tag file as text to the standard output. Output format is
-		'key<tab>data'. This is for debugging.
+		\'key<tab>data'. This is for debugging.
 	@item{@option{--explain}}
 		Explain handling files.
 	@item{@option{-f}, @option{--file} @arg{file}}
