From: Punit Agrawal <punitagrawal@gmail.com>
Date: Tue, 25 Mar 2014 18:45:47 +0000
Subject: Fix gtags.el to not use goto-line

---
 gtags.el | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: global-6.5.5/gtags.el
===================================================================
--- global-6.5.5.orig/gtags.el
+++ global-6.5.5/gtags.el
@@ -875,7 +875,8 @@
 	    (find-file-other-window file)))
         (if delete (kill-buffer prev-buffer)))
       (setq gtags-current-buffer (current-buffer))
-      (goto-line line)
+      (goto-char (point-min))
+      (forward-line (1- line))
       (gtags-mode 1))))
 
 ;; make complete list (do nothing)
